src/app/admin/collectionCentresTable/CollectionCentresTable.tsx

```diff
@@ -250,6 +364,31 @@ const CollectionCentresTable: React.FC = () => {
             editable: true,
             renderHeader: (params) => <Header {...params} />,
         },
+        {
+            field: "collectionSlot",
+            type: "actions",
+            headerName: "Collection Slots",
+            flex: 1,
+            renderHeader: (params) => <Header {...params} />,
+            renderCell: (params) => {
```

Comment:
Just trying to understand the code - do you know where it gets these params from and how does it know the row primary key?

Reply:
Not 100% but my understanding is that the rows get passed into DataGrid and when an action button is clicked on that row it passed which row on to the params. Row is defined as type CollectionCentresTableRow and the primary key is params.row.id.